ai_genomics/pipeline/README.md

```diff
@@ -44,3 +44,11 @@ The AI and genomics Crunchbase organisation ids can be loaded using `ai_genomics
 Run `python ai_genomics/analysis/gtr_definitions.py` to print a summary of AI / genomics projects in the GtR data. This also saves a list of project in the intersection of AI and genomics in `inputs/data/gtr/gtr_ai_genomics_projects.json`.
 
 The AI and genomics GtR projects and publications can be loaded using `ai_genomics.getters.gtr.get_ai_genomics_gtr_data`.
+
+## Validation
+
+This pipeline produces samples of data for expert validation. Specifically, it produces samples of patent and publication abstracts and the OpenAlex concepts and CPC patent codes used to generate those datasets. The output samples vary in format. The samples for patent and publication abstracts are partially overlapping, such that each validation contributor assesses a small sample of identical abstracts and a larger number of different ones. This is designed to maximise the breadth of coverage, while gathering information to calculate inter-contributor agreement. The CPC patent code samples are also overlapping samples but the OpenAlex concepts are provided in full as there are so few.
+
+First, run `python ai_genomics/pipeline/patent_data/cpc_lookup.py` to create a lookup of CPC codes, including information about their parent codes.
+
+Next, run `python ai_genomics/pipeline/validation/abstracts.py` and `python ai_genomics/pipeline/validation/definitions.py` to produce the validation outputs. These must then be formatted in a spreadsheet for subject matter expert contributions.
```

Comment:
i was curious to see if you could just do this programmatically - apparently there's a library called gspread, a python API for google sheets - and you can push a df to sheets using this library.  

it would be absolutely overkill as you would need to generate credentials and that alone would take more time than just manually importing the .csv to a spreadsheet but there ya go!

Reply:
Very good to know! Probably won't use it now, but could see a world where that could be useful. Especially if it can also pull data in?